pydrive2/test/test_oauth.py

```diff
@@ -3,10 +3,15 @@
 import time
 
 from pydrive2.auth import GoogleAuth
+from pydrive2.test import test_util
 
-class GoogleAuthTest(unittest.TestCase):
+class GoogleAuthTest(test_util.PyDriveTestBase):
   """Tests basic OAuth2 operations of auth.GoogleAuth."""
 
+  def setUp(self):
+    if os.getenv(test_util.PyDriveTestBase.GDRIVE_USER_CREDENTIALS_DATA):
+      self.skipTest('OAuth test is skipped. Unset {} env var.'.format(test_util.PyDriveTestBase.GDRIVE_USER_CREDENTIALS_DATA))
+
```

Comment:
I probably don't understand the whole logic behind test1 - test6 files, but should we use test6 for ServiceAuth ?

Reply:
why do we bother here at all if we just skip these tests anyway? is there a way to run them?

Reply:
Have removed this check completely.
New test for ServiceAuth() and test6.yaml is added.
`@pytest.mark.automated` custom mark is used to run tests on Travis.